tls-client/easy-connect.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/easy-connect/#5b52b5fa56c623d5853c5daf266b34986a0c20cc
+https://github.com/ARMmbed/easy-connect/#bf48f5b649eba3ed7b810c77c630b5eb51301311
```

Comment:
Wouldn't it work if we change to https://github.com/ARMmbed/easy-connect/tree/v1.2.16 ?
I think it will be more readable this way

Reply:
Mbed cli tooling does not support that, unfortunately.